### ISSUE TITLE
[Bug]: Error on CIFS service setup

### ISSUE THREAD

**Issue by dsnelleman** [NetApp/terraform-provider-netapp-ontap#250]
### Terraform Core Version

1.9.3

### ONTAP Provider Version

1.1.2

### Affected Resource(s)

netapp-ontap_protocols_cifs_service_resource
netapp-ontap_protocols_cifs_service_data_source
netapp-ontap_protocols_cifs_services_data_source

### Expected Behavior

Configure CIFS service on SVM

### Actual Behavior

plan fails

### Relevant Error/Panic Output Snippet

```shell
Error: Invalid Path Expression for Schema
│
│   with netapp-ontap_protocols_cifs_service_resource.svmtest_cifs,
│   on cifs.tf line 1, in resource "netapp-ontap_protocols_cifs_service_resource" "svmtest_cifs":
│    1: resource "netapp-ontap_protocols_cifs_service_resource" "svmtest_cifs" {
│
│ The Terraform Provider unexpectedly provided a path expression that does not match the current schema. This can happen if the path expression does not correctly follow the schema in structure or types. Please
│ report this to the provider developers.
│
│ Path Expression: advertised_kdc_encryptions
╵
```

### Terraform Configuration Files

resource "netapp-ontap_protocols_cifs_service_resource" "svmtest_cifs" {
  ad_domain = {
    fqdn                = "testlab.mylocal"
    organizational_unit = "CN=Computers"
    password            = var.DC_password
    user                = var.DC_username
  }
  comment           = null
  cx_profile_name   = "SCLTEST-01"
  default_unix_user = "pcuser"
  enabled           = true
  force             = null
  name              = "svmtest"
  netbios = {
    aliases      = null
    enabled      = false
    wins_servers = []
  }
  security = {
    advertised_kdc_encryptions = ["aes_128","aes_256","des","rc4"]
    aes_netlogon_enabled       = false
    encrypt_dc_connection      = false
    kdc_encryption             = true
    ldap_referral_enabled      = false
    lm_compatibility_level     = "lm_ntlm_ntlmv2_krb"
    restrict_anonymous         = "no_enumeration"
    session_security           = "none"
    smb_encryption             = false
    smb_signing                = false
    try_ldap_channel_binding   = true
    use_ldaps                  = false
    use_start_tls              = false
  }
  svm_name = "svmtest"
}

### Steps to Reproduce

1. Build an svm
2. build a AD
3. change variables to your config
4. plan your terraform config 

### Panic Output

 Error: Invalid Path Expression for Schema
│
│   with netapp-ontap_protocols_cifs_service_resource.svmtest_cifs,
│   on cifs.tf line 1, in resource "netapp-ontap_protocols_cifs_service_resource" "svmtest_cifs":
│    1: resource "netapp-ontap_protocols_cifs_service_resource" "svmtest_cifs" {
│
│ The Terraform Provider unexpectedly provided a path expression that does not match the current schema. This can happen if the path expression does not correctly follow the schema in structure or types. Please
│ report this to the provider developers.
│
│ Path Expression: advertised_kdc_encryptions
╵

### Would you like to implement a fix?

None

**Comment by chuyich** [comment]
If you use advertised_kdc_encryptions which is introduced in 9.12, then kdc_encryption cannot be used cause this attribute has been deprecated. You may remove kdc_encryption from the resource file to unblock your execution. Will make adjustment and failed on the ONTP version check.

**Comment by dsnelleman** [comment]
> If you use advertised_kdc_encryptions which is introduced in 9.12, then kdc_encryption cannot be used cause this attribute has been deprecated. You may remove kdc_encryption from the resource file to unblock your execution. Will make adjustment and failed on the ONTP version check.

That worked, Tnx

**Comment by chuyich** [comment]
The fix will be in next release 1.1.3
